.kilo/plans/1781937301606-parity-mod-visibility-regex.md

```
@@ -0,0 +1,69 @@
+# Fix `mod` traversal regex for visibility-qualified declarations (parity baseline generators)
+
+## Goal
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)